scripts/src/main/resources/scripts/command/az

```diff
@@ -0,0 +1,151 @@
+      modifyPathStructure
+      installOnWindows
+    else doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
+    fi
+  fi
+}
+
+# modify a filepath for powershell path syntax e.g. /c/... -> c:/...
+function modifyPathStructure(){
+  modifiedPath=${DEVON_DOWNLOAD_DIR}
+  partition=$(echo "${modifiedPath}" | cut -f2 -d'/'):'/' # -> <partition>:/
+  modifiedPath=${partition}$(echo "${modifiedPath}" | cut -f3- -d'/')
+  modifiedPath="${modifiedPath////\\}"
+}
+
+# Install Azure CLI for Windows via Powershell Command
+function installOnWindows(){ 
+  if doIsPwshInstalled
+  then
+    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${modifiedPath}\az-${version}-windows.msi /QB-!'" &> /dev/null || exit_code=$?
+    if [[ -z "${exit_code}" ]]
+    then 
+      setConfigDir
+      doEcho "Azure CLI has been installed and is ready to use. \n\n******** IMPORTANT ******** \nRestart the console to use Azure CLI regular." 
+    else
+      read -r -p "You must run the installation with administration rights. Would you like to start the installation again? (Y/n)" answer 
+      case ${answer:0:1} in 
+        y|Y )
+          doEcho "\nInstalling Azure CLI for Windows..."
+          exit_code=""
+          installOnWindows
+        ;;
+        * )
+          doEcho "The installation has been canceled."
+        ;;
+      esac
+    fi
+  else doEcho "Powershell must be installed on your system to continue the installation."
+  fi
+}
+    
+
+# Set directory for azure cli configuration file
+function setConfigDir() {
+  azureconf_dir="${DEVON_IDE_HOME}/conf/.azure"
+  azurecli_export="export AZURE_CONFIG_DIR=${azureconf_dir}"
+  if ! grep -q "${azurecli_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      doRunCommand "${azurecli_export}"
+      echo -e "\n${azurecli_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+      doEcho "Location of Azure's configuration file is set to ${azureconf_dir}"
+  fi
+}
+
+# OS independet check if Azure CLI is installed
+function doIsAzureInstalled() {
+  if command -v az &> /dev/null
+  then 
+    return
+  else
+    return 255
+  fi
+}
+
+# Check if Powershell is installed
+function doIsPwshInstalled() {
+  if command -v powershell &> /dev/null
+  then
+    return
+  else
+    return 255
+  fi
+}
+
+function doRun() {
+  doSetup silent
+  doEcho "Running: Azure CLI ${*}"
+  setConfigDir
+  az "${@}"
+}
+
+# CLI
+case ${1} in
+"help" | "-h")
+  echo "Install Azure CLI."
+  echo
+  echo "Arguments:"
+  echo " setup                    install Azure CLI on your machine (global installation)."
+  echo " <<args>>                 call Azure CLI with the specified arguments. Call az --help for details or use Azure CLI directly as preferred."
+  echo
+;;
+
+"setup" | "s")
+  doEcho "Installing Azure CLI..."
+  doSetup
```

Comment:
you should pass the arguments (e.g. "silent" option) like in other commandlets as you currently losing this here.